tck/features/MergeNodeAcceptance.feature

```diff
@@ -524,7 +702,12 @@ Feature: MergeNodeAcceptance
 
   Scenario: Merge inside foreach should see variables introduced by update actions outside foreach
     Given an empty graph
-    When executing query: CREATE (a {name: 'Start'}) FOREACH(x IN [1,2,3] | MERGE (a)-[:X]->({id: x})) RETURN a.name
+    When executing query:
+      """
+      CREATE (a {name: 'Start'})
+      FOREACH(x IN [1,2,3] | MERGE (a)-[:X]->({id: x}))
```

Comment:
Just had a discussion about this, and yes - let's keep this consistent. So, a space after a comma in an sequence, whether it is a list, map, proc args....